AmplifyPlugins/API/Sources/AWSAPIPlugin/AppSyncRealTimeClient/AppSyncRealTimeClient.swift

```diff
@@ -54,6 +54,10 @@ actor AppSyncRealTimeClient: AppSyncRealTimeClientProtocol {
         self.state.value == .connected
     }
 
+    internal var numOfSubscriptions: Int {
```

Comment:
[nit] a slight preference for spelling the entire word out `numberOfSubscriptions`

Reply:
thanks, will update the name